tests/e2e/utils.go

```diff
@@ -90,3 +97,21 @@ func getActiveApplication() string {
 	odoActiveApp := strings.Split(reActiveApp.FindString(stdOut), "*")[1]
 	return strings.TrimSpace(odoActiveApp)
 }
+
+func retryingForOutputMatchStringOfHTTPResponse(url, match string, retry, sleep int) bool {
+	for i := 0; i < retry; i++ {
```

Comment:
If variable `i` is not being used
use simple for loop without variable, like this
```
for {
}
```

Reply:
It is being used in the loop itself, as the counter.